src/redux/uniswapLiquidity.js

```diff
@@ -58,6 +60,9 @@ export const uniswapUpdateLiquidityTokens = (
       ),
       token => !!Number(token?.balance?.amount ?? 0)
     );
+  } else {
+    const assetCodes = map(liquidityTokens, token => token.address);
```

Comment:
This is only for a fallback scenario right? Shouldn't be too many I guess

Reply:
It is also for "received" LP tokens (so any fresh asset address received whether thru Zerion or fallback explorer will trigger the charts request for LP tokens)